pubsub/api/QuickStart/QuickStart.cs

```diff
@@ -22,16 +22,19 @@ namespace GoogleCloudSamples
 {
     public class QuickStart
     {
-        public static void Main(string[] args)
+        public static int Main(string[] args)
         {
             // Instantiates a client
             PublisherServiceApiClient publisher = PublisherServiceApiClient.Create();
 
             // Your Google Cloud Platform project ID
             string projectId = "YOUR-PROJECT-ID";
             // [END pubsub_quickstart]
-            Debug.Assert(projectId != "YOUR-PROJECT" + "-ID",
-                "Edit Program.cs and replace YOUR-PROJECT-ID with your project id.");
+            if (projectId == "YOUR-PROJECT" + "-ID")

```

Comment:
I like it but it looks like not all the samples follow this pattern. Should we fix all of them later on so all of them are consistent?